spec/routes/web/clover_web_spec.rb

```diff
@@ -117,4 +117,18 @@
 
     expect(failures).to be_empty
   end
+
+  it "returns ipv4 list from cached file" do
+    vmh1, vmh2 = Array.new(2) { create_vm_host }
+    Address.create(cidr: "1.1.1.1/32", routed_to_host_id: vmh1.id) { it.id = vmh1.id }
+    Address.create(cidr: "3.3.3.3/27", routed_to_host_id: vmh2.id)
+    Address.create(cidr: "2.2.2.2/27", routed_to_host_id: vmh1.id)
+    Address.create(cidr: "3.3.3.3/28", routed_to_host_id: vmh2.id)
```

Comment:
Probably should have an IP address like `10.10.10.10/28` so you can test the sorting works as expected.